inventory

```diff
@@ -0,0 +1,3 @@
+[dev-machines]
+
+localhost   ansible_connection=local
```

Comment:
So one inventory file should have the host with connection local and the other inventory should point to the VM over SSH.

Reply:
Okay, but I need an example